include/pclbo/pclbo.h

```diff
@@ -2,6 +2,7 @@
 #define PC_LBO_DIFFUSION_HH
 
 #include <cmath>
+#include <boost/thread/thread.hpp>
```

Comment:
why is it necessary to include boost's thread lib?

Reply:
I confirm,  the boost thread is not necessary if pthread is present! My bad!